src/simulation/dynamics/spinningBodies/spinningBodiesOneDOF/_UnitTest/test_spinningBodyOneDOFStateEffector.py

```diff
@@ -65,11 +63,15 @@ def test_spinningBodyTest(show_plots):
 
     against their initial values.
     """
-    [testResults, testMessage] = spinningBody(show_plots)
+    [testResults, testMessage] = test_spinningBody(show_plots, cmdTorque, lock)
```

Comment:
undo this change.  Call the `spinningBody` method here with the `cmdTorque` and `lock` variables

Reply:
Done.